docs/NMOS With H.264.md

```diff
@@ -58,11 +58,116 @@ The term 'strict-CBR' corresponds to the definition of constant bit-rate at sect
 The terms CBR and VBR are defined in the [ITU-T H.264][H.264] | ISO/IEC 14496 standard.
 
 ## H.264 IS-04 Sources, Flows and Senders
+
+Nodes capable of transmitting H.264 video streams MUST have Source, Flow and Sender resources in the IS-04 Node API.
+
 ### Sources
+
+The Source resource MUST indicate `urn:x-nmos:format:video` for the `format` attribute. 
+- Source resources MAY be associated with many Flows at the same time. The Source is therefore unaffected by the use of H.264 compression.
+
 ### Flows
+
+The Flow resource MUST indicate `video/H264` in the `media_type` attribute, and `urn:x-nmos:format:video` for the `format` attribute. This has been permitted since IS-04 v1.1. 
+- H.264 Flow resources MAY be associated with many Senders at the same time through the Senders' `flow_id` attributes. The H.264 Flow is therefore unaffected by the use of a specific transport. 
+- H.264 Flow resources MAY be associated with many Flows at the same time through the Flows' `parents` attributes. The H.264 Flow is therefore unaffected by being the parent of some other Flows.
+
+For Nodes implementing IS-04 v1.3 or higher, the following additional requirements on the Flow resource apply.
+
+In addition to those attributes defined in IS-04 for all coded video Flows, the following attributes defined in the [Flow Attributes Register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/) of the [NMOS Parameter Registers][] are used for H.264.
+
+These attributes provide information for Controllers and Users to evaluate stream compatibility between Senders and Receivers.
+
+- [Components](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#components)
+  The Flow resource MUST indicate the color (sub-)sampling, width, height and depth of the associated uncompressed picture using the `components` attribute. The `components` array values MUST correspond to the stream's active parameter sets values. A Flow MUST track the stream's current active parameter sets.
+
+Informative note: ST 2110-22 does not require the `sampling` or `depth` SDP parameters. RFC 6184 does not define any such SDP parameters. The `sampling` and `depth` of the associated uncompressed picture could be derived from the H.264 active parameter sets by a Receiver.
+
+- [Profile](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#profile)
+  The Flow resource MUST indicate the H.264 profile, which defines algorithmic features and limits that SHALL be supported by all decoders conforming to that profile. It SHALL comply with the stream's active parameter sets. The permitted `profile` values are strings, defined as per ITU-T  Rec. H.264 Annex A
```

Comment:
Not sure about "comply with"... would "correspond to" or "be consistent with" be better?

Reply:
The word "corresponds" is too strict as the Flow may indicate a profile A that as per the H.264 specification implies support for profiles B and C ... and the active parameter set may indicate profile B at a given time ... This is why I used the word "comply" because it means that the Flow must indicate a profile that is compliant as per the H.264 standard to the actual profile definition of the stream.

Reply:
So you are saying one profile can be considered compliant with another profile... OK. But then I think you want the opposite, you wanted to say that the active picture parameter set value must be the same or lower than the one in the Flow, right?

Reply:
There reverse would also be true ... The profile of the Flow must be higher or the same than the one of the active picture parameter set value ... 

Instead of saying "higher or the same" or "lower or the same"  ... which is easier for level than profile ... I tried to use the word "compliant" or "comply" ...  I'm not sure of the normative meaning of "consistent" versus "compliant"

Would this be better: The stream's active parameter sets MUST be compliant with the profile attribute of the Flow.

And we would have a similar one for the level: The stream's active parameter sets MUST be compliant with the level attribute of the Flow.

Reply:
Those make more sense to me at least.

Reply:
Ok, will go with this for now and possibly get back to it later.